src/modules/projects/hooks/useProjectDashboardNavItems.tsx

```diff
@@ -105,9 +109,11 @@ export const useProjectDashboardNavItems = (
               'canViewUnits',
               'canViewReferrals',
               'canViewOwnReferrals',
+              'canManageOutgoingReferrals',
             ],
             permissionMode: 'any',
```

Comment:
```suggestion
```
Can we remove these now? or are they checking something additional beyond what useProjectCeVisibility checks? It would be nice not to have to maintain this list of permissions.

Reply:
Looks like yes they can be removed. I can do some quick QA and open another PR.